metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -52,6 +52,26 @@ def make_data_job_urn(
     )
 
 
+def make_ml_primary_key_urn(feature_table_name: str, primary_key_name: str) -> str:
```

Comment:
Having to add this for each type is against the nocode principle. @jjoyce0510 @hsheth2 Any ideas to prevent this in the future?

For this PR, I think it's okay.

Reply:
Right - I think in the long term we want to permit production of key objects instead of urns on the ingest path. Alternatively, we can could do some auto-generation of the Python code from the PDL model. For now, we'll have to do this.